x-pack/plugins/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -67,300 +128,370 @@ export const DeploymentSetup: FC<DeploymentSetupProps> = ({
+        />
+      ),
+      'data-test-subj': `mlModelsStartDeploymentModalOptimizedForSearch`,
+    },
+  ];
+
+  const helperText = useMemo<string>(() => {
+    const vcpuRange = deploymentParamsMapper.getVCPURange(config.vCPUUsage);
+
+    if (config.adaptiveResources && cloudInfo.isMlAutoscalingEnabled) {
+      switch (config.vCPUUsage) {
+        case 'low':
+          return i18n.translate(
+            'xpack.ml.trainedModels.modelsList.startDeployment.lowCpuAdaptiveHelp',
+            {
+              defaultMessage:
+                'This level limits resources to the minimum required for ELSER to run if supported by your Cloud console selection. It may not be sufficient for a production application.',
+            }
+          );
+        case 'medium':
+          return i18n.translate(
+            'xpack.ml.trainedModels.modelsList.startDeployment.mediumCpuAdaptiveHelp',
+            {
+              defaultMessage:
+                'Your model will scale up to a maximum of {maxVCPUs} vCPUs. Even if the Cloud console provides more, the model will not exceed {maxVCPUs}, leaving additional resources for other models.',
+              values: { maxVCPUs: vcpuRange.max },
+            }
+          );
+        case 'high':
+          return i18n.translate(
+            'xpack.ml.trainedModels.modelsList.startDeployment.highCpuAdaptiveHelp',
+            {
+              defaultMessage:
+                'Your model may scale up to the maximum number of vCPUs available for this deployment from the Cloud console if needed. If the maximum is {minVCPUs} vCPUs or fewer, this level is equivalent to the medium level.',
+              values: { minVCPUs: vcpuRange.min },
+            }
+          );
+      }
+    } else {
+      switch (config.vCPUUsage) {
+        case 'low':
+          return i18n.translate(
+            'xpack.ml.trainedModels.modelsList.startDeployment.lowCpuStaticHelp',
+            {
+              defaultMessage:
+                'This level limits resources to one vCPU, which may be suitable for development, testing, and demos depending on your parameters. It is not recommended for production use.',
+            }
+          );
+        case 'medium':
+          return i18n.translate(
+            'xpack.ml.trainedModels.modelsList.startDeployment.mediumCpuStaticHelp',
+            {
+              defaultMessage:
+                'This level limits resources to two vCPUs, which may be suitable for development, testing, and demos depending on your parameters. It is not recommended for production use.',
+            }
+          );
+        case 'high':
+          return i18n.translate(
+            'xpack.ml.trainedModels.modelsList.startDeployment.highCpuStaticHelp',
+            {
+              defaultMessage:
+                'This level may use the maximum number of vCPUs available for this deployment from the Cloud console. If the maximum is 2 vCPUs or fewer, this level is equivalent to the medium or low level.',
```

Comment:
Should this message for high vCPUs level refer to the cloud console when running locally?